### PR TITLE
Admin Guide: Add doccument to export Gluster volume via SMB protocol

### PR DESCRIPTION
**Pull request by poornimag**
Signed-off-by: Poornima G <pgurusid@redhat.com>

### CODE REVIEW

File: Administrator Guide/Accessing Gluster from Windows.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Accessing Gluster volume via SMB Protocol
+
+Layered product Samba is used to export the Gluster volum and ctdb for providing the high availability Samba.
```
Comment (prashanthpai) [inline Administrator Guide/Accessing Gluster from Windows.md]: volume

Comment (raghavendra-talur) [inline Administrator Guide/Accessing Gluster from Windows.md]: s/volum/volume

File: Administrator Guide/Accessing Gluster from Windows.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+#####Step 2: Enable/Disable the auto export of Gluster volume via SMB
+```# gluster volume set user.smb disable/enable```
+
+#####Step 3: Setup the CTDB Cluster:
+ 1. Create a ctdb meta volume with replica N, N being the number of the servers that are used as Samba servers.
+   This volume will host only a zero byte lock file, hence choose the minimal sized bricks.
+   To create the n replica volume run the following command:  
+   ```# gluster volume create <volname> replica n <ipaddr/host name>:/<brick_patch>.... N times```
+
+ 2. In the following files, replace "all" in the statement "MENTA=all" to the newly created volume name.
```
Comment (prashanthpai) [inline Administrator Guide/Accessing Gluster from Windows.md]: META

File: Administrator Guide/Accessing Gluster from Windows.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    /var/lib/glusterd/hooks/1/stop/pre/S29CTDB-teardown.sh
+    ```
+
+ 3. Start the ctdb volume  
+   ```# gluster vol start <volname>```
+
+ 4. Verify the following:
+    * If the following lines are added in smb.conf file in all the nodes running samba/ctdb:  
+    `
+    clustering = yes
```
Comment (prashanthpai) [inline Administrator Guide/Accessing Gluster from Windows.md]: These are rendered in the same line

**Comment by poornimag** [comment]
Comments addressed

**Pull request merged**